### DISCUSSION TITLE
Import Historic Data from ST2 to InfluxDB

### DISCUSSION THREAD

**Discussion by UncommonGarb** [alexjustesen/speedtest-tracker#329] [Ideas]
Currently I have data in ST2 and just setup influxdb. There is not a way that I see to force ST2 to send historic data to influxdb to update it, it only pushes from the start of the influx db. 

Thank you for your work.

**Comment by alexjustesen** [comment]
At the moment there isn't, how far back do you push data to influxdb for the purposes of dashboard?
